aten/src/ATen/native/native_functions.yaml

```diff
@@ -446,6 +446,7 @@
   dispatch:
     CPU: _s_copy__cpu
     CUDA: _s_copy__cuda
+    QuantizedCPU: _s_copy__quantized
```

Comment:
Just wondering, will we have to support copy between two different quantized tensors?